src/BoxColor.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+
+export default function BoxColor(props) {
```

Comment:
podemos desestructurar aqui el contenido del props y nos ahorramos la linea  5 👍 
```suggestion
export default function BoxColor({ r, g, b, text }) {
```